lib/vsc/install/shared_setup.py

```diff
@@ -524,9 +616,18 @@ def sanitize(name):
 
 
 def parse_target(target):
-    """Add some fields"""
+    """
+    Add some fields
+
+    Remove sdist vsc class with '"vsc_sdist": False' in target
+    """
     new_target = {}
     new_target.update(SHARED_TARGET)
+
+    vsc_sdist = target.pop('vsc_sdist', True)
+    if not vsc_sdist:
+        new_target['cmdclass'].pop('sdist')
```

Comment:
do you need to check whether `sdist` is something else than the `vsc_sdist` class?